PRODUCTION.md

```diff
@@ -0,0 +1,82 @@
+# Using S3 for continuous integration testing with Docker
+
+When you start the Scality S3server image, you can adjust the configuration of
+the Scality S3server instance by passing one or more environment variables
+on the docker run command line.
+
+## Environment Variables
+
+### HOST_NAME
+
+This variable specifies an host name:
+
+`docker run -d --name s3server -p 8000:8000 -e HOST_NAME=new.host.com
+scality/s3server`
+
+### ACCESS_KEY and SECRET_KEY
+
+These variables specify a Scality S3server user credentials named Docker:
+
+`docker run -d --name s3server -p 8000:8000 -e ACCESS_KEY=newAccessKey -e
+SECRET_KEY=newSecretKey scality/s3server`
+
+### LOG_LEVEL
+
+This variable allows you to change the log level: info, debug or trace:
```

Comment:
The default is info.  Debug will give you more detailed logs and trace will give you the most detailed.